server/services/dashboard/dashboard.go

```diff
@@ -88,6 +98,15 @@ func (t *dashboard) getDashboardGrid() ([][]*schema.DashboardElementView, error)
 				if err != nil {
 					return nil, err
 				}
+			case schema.DashboardElementTypeTCBuilds:
+				tcBuildInfo, err := t.sideStats.TCBuildInfo()
+				if err != nil {
+					return nil, err
+				}
+
+				elementView.BuildStatuses = tcBuildInfo.BuildStatuses
+				elementView.LastChangesAuthor = types.NewNullString(tcBuildInfo.LastChangesAuthor)
```

Comment:
Ok. The point I've done so was that I've asked @lisitsky what actually to show and he told me 

> Doesn't matter. For instance, show statuses of the last 5 builds and author and date of the last changes.